org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeChangedFilesJob.java

```diff
@@ -59,28 +54,30 @@ public IStatus runInWorkspace(IProgressMonitor monitor) {
     try {
       global.setTaskName("Collect changed file(s) list");
       Arrays.asList(ResourcesPlugin.getWorkspace().getRoot().getProjects()).stream().forEach(MarkerUtils::deleteChangeSetIssuesMarkers);
-      Collection<IFile> collectChangedFiles = collectChangedFiles(projects, global.newChild(20));
+      Collection<ISonarLintFile> collectChangedFiles = collectChangedFiles(projects, global.newChild(20));
 
       if (collectChangedFiles.isEmpty()) {
         SonarLintLogger.get().info("No changed files found");
         return Status.OK_STATUS;
       }
 
-      Map<IProject, Collection<IFile>> changedFilesPerProject = SonarLintUtils.aggregatePerMoreSpecificProject(collectChangedFiles);
+      // FIXME duplicate Map<ISonarLintProject, Collection<ISonarLintFile>> changedFilesPerProject =
+      // SonarLintUtils.aggregatePerMoreSpecificProject(collectChangedFiles);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)